integration-test/rollingupdate-kubernetes/kubernetes/akka-cluster-app-value-revision.yml

```diff
@@ -65,8 +65,11 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: pod-reader
 rules:
-- apiGroups: ["apps", ""] # additional access to api to get revision
-  resources: ["pods", "replicasets"]
+- apiGroups: [""]
+  resources: ["pods"] # access to get replicaset
+  verbs: ["get", "list"]
+- apiGroups: ["apps"]
+  resources: ["replicasets"] # access to get revision annotation
```

Comment:
it works to define it as `["apps", ""]` but it's rather confusing

Reply:
indeed, better now